manifests/r/RealVNC/VNCServer/6.9.0.46599/RealVNC.VNCServer.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RealVNC.VNCServer
+PackageVersion: 6.9.0.46599
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn REBOOT=ReallySuppress
+  SilentWithProgress: /qn REBOOT=ReallySuppress
+UpgradeBehavior: install
```

```suggestion
UpgradeBehavior: install
ElevationRequirement: elevationRequired
```
Requires CMD to be run as Administrator